`make gcloud-auth-docker` fails on Windows

Top level bug: #47 

<details>
<summary>
<code>markmandel@DESKTOP-BDM5UCP:/c/Users/Mark/Documents/workspace/agon/build$ make gcloud-auth-docker</code>
</summary>
<pre>
mkdir -p /c/Users/Mark/Documents/workspace/agon/build//.kube
mkdir -p /c/Users/Mark/Documents/workspace/agon/build//.config/gcloud
sudo rm -rf /tmp/gcloud-auth-docker
mkdir -p /tmp/gcloud-auth-docker
cp ~/.dockercfg /tmp/gcloud-auth-docker
cp: cannot stat '/home/markmandel/.dockercfg': No such file or directory
Makefile:222: recipe for target 'gcloud-auth-docker' failed
make: [gcloud-auth-docker] Error 1 (ignored)
docker run --rm -v /c/Users/Mark/Documents/workspace/agon/build//.config/gcloud:/root/.config/gcloud -v ~/.kube:/root/.kube -v /c/Users/Mark/Documents/workspace/agon:/go/src/github.com/agonio/agon -v /tmp/gcloud-auth-docker:/root --entrypoint="gcloud" agon-build:6c2ef6cd74 docker --authorize-only
Short-lived access for ['gcr.io', 'us.gcr.io', 'eu.gcr.io', 'asia.gcr.io', 'l.gcr.io', 'launcher.gcr.io', 'us-mirror.gcr.io', 'eu-mirror.gcr.io', 'asia-mirror.gcr.io', 'mirror.gcr.io', 'k8s.gcr.io'] configured.
sudo mv /tmp/gcloud-auth-docker/.dockercfg ~/
mv: cannot stat '/tmp/gcloud-auth-docker/.dockercfg': No such file or directory
Makefile:222: recipe for target 'gcloud-auth-docker' failed
make: *** [gcloud-auth-docker] Error 1
</pre>
</details>

/cc @Kuqd

--- a/.gitignore
+++ b/.gitignore
@@ -17,3 +17,4 @@
 *.iml
 bin
 *.o
+tmp

--- a/build/Makefile
+++ b/build/Makefile
@@ -219,11 +219,11 @@ gcloud-auth-cluster: ensure-build-image
 # authenticate our docker configuration so that you can do a docker push directly
 # to the gcr.io repository
 gcloud-auth-docker: ensure-build-image
-	-sudo rm -rf /tmp/gcloud-auth-docker
-	mkdir -p /tmp/gcloud-auth-docker
-	-cp ~/.dockercfg /tmp/gcloud-auth-docker
-	docker run --rm $(common_mounts) -v /tmp/gcloud-auth-docker:/root --entrypoint="gcloud" $(build_tag) docker --authorize-only
-	sudo mv /tmp/gcloud-auth-docker/.dockercfg ~/
+	-sudo rm -rf $(build_path)/tmp
+	mkdir -p $(build_path)/tmp/gcloud-auth-docker
+	-cp ~/.dockercfg $(build_path)/tmp/gcloud-auth-docker
+	docker run --rm $(common_mounts) -v $(build_path)/tmp/gcloud-auth-docker:/root $(build_tag) gcloud docker --authorize-only
+	sudo mv $(build_path)/tmp/gcloud-auth-docker/.dockercfg ~/
 	sudo chown $(USER) ~/.dockercfg
 
 # Clean the gcloud configuration

